### DISCUSSION TITLE
Password policy regex message customization

### DISCUSSION THREAD

**Discussion by gabardyne** [keycloak/keycloak#31865] [Q&A]
Hello,

I want to know if it is possible to customize the message when the password don't meet the regex password policy.

For now, the message is "Mot de passe invalide : ne valide pas l'expression rationnelle." I think in english, it must be something like "Invalid password : Don't respect the regular expression".

Regards,
Gab

**Comment by darius-m** [comment]
It is possible, but you'll have to create a custom theme and override the message - https://www.keycloak.org/docs/latest/server_development/index.html#_themes. You can define just the message that you want to override, and the others will be retrieved from the parent theme automatically.

**Comment by gabardyne** [comment]
Hello,
Thank you for the answer, I'll investigate that.
Regards,
Gab
